src/cli/flags/export.go

```diff
@@ -0,0 +1,15 @@
+package flags
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const ArchiveFN = "archive"
+
+var ArchiveFV bool
+
+// AddExportConfigFlags adds the restore config flag set.
+func AddExportConfigFlags(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.BoolVar(&ArchiveFV, ArchiveFN, false, "Archive contents")
```

Comment:
This needs a better description.  `--archive | Archive Contents` is kinda like explaining: `--foo | Foo stuff`.

Reply:
Changed to `Export data as an archive instead of individual files`.